crates/frontend/src/pages/config_version.rs

```diff
@@ -9,11 +9,12 @@ use leptos_router::use_params_map;
 pub fn config_version() -> impl IntoView {
     let params = use_params_map();
     let tenant = params.with(|p| p.get("tenant").cloned().unwrap_or_default());
+    let org_id = params.with(|p| p.get("org_id").cloned().unwrap_or_default());
```

Comment:
shouldn't we be using this here ?
```suggestion
    let tenant_rws = use_context::<RwSignal<Tenant>>().unwrap();
    let org_rws = use_context::<RwSignal<OrganisationId>>().unwrap();
```